2022-07-26 10:53:39,475 - qtpyvcp - INFO - Logging to /home/travis/linuxcnc/configs/my_LinuxCNC_router/qtpyvcp.log
2022-07-26 10:53:39,477 - qtpyvcp - INFO - QtPyVCP Version: python3_master+273.g268f4593
2022-07-26 10:53:40,382 - qtpyvcp.utilities.info - INFO - The machine has 3 axes and 3 joints
2022-07-26 10:53:40,383 - qtpyvcp.utilities.info - INFO - The Axis/Joint mapping is:
2022-07-26 10:53:40,383 - qtpyvcp.utilities.info - INFO - Axis X --> Joint 0
2022-07-26 10:53:40,390 - qtpyvcp.utilities.info - INFO - Axis Y --> Joint 1
2022-07-26 10:53:40,390 - qtpyvcp.utilities.info - INFO - Axis Z --> Joint 2
2022-07-26 10:53:40,429 - qtpyvcp - INFO - QtPyVCP Version: python3_master+273.g268f4593
2022-07-26 10:53:41,682 - qtpyvcp.utilities.info - WARNING - No [TRAJ] SPINDLES entry in INI, using 1
2022-07-26 10:53:41,682 - qtpyvcp.utilities.info - WARNING - No [DISPLAY] DEFAULT_SPINDLE_SPEED entry found in INI, using 300rpm
2022-07-26 10:53:42,099 - qtpyvcp - INFO - Loading global stylesheet: /home/travis/.local/lib/python3.9/site-packages/probe_basic/probe_basic.qss
2022-07-26 10:53:48,249 - qtpyvcp.widgets.form_widgets.main_window - ERROR - Error binding menu action settings.backplot.show-program-labels
Traceback (most recent call last):
  File "/home/travis/.local/lib/python3.9/site-packages/qtpyvcp/actions/__init__.py", line 61, in bindWidget
    method = getattr(method, item)
AttributeError: module 'qtpyvcp.actions' has no attribute 'settings'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/travis/.local/lib/python3.9/site-packages/qtpyvcp/widgets/form_widgets/main_window.py", line 185, in getMenuAction
    actions.bindWidget(menu_action, action_name)
  File "/home/travis/.local/lib/python3.9/site-packages/qtpyvcp/actions/__init__.py", line 66, in bindWidget
    raise InvalidAction("Could not get action method: %s" % item)
qtpyvcp.actions.InvalidAction: Could not get action method: settings
2022-07-26 10:53:48,251 - qtpyvcp.widgets.form_widgets.main_window - ERROR - Error binding menu action settings.backplot.show-program-ticks
Traceback (most recent call last):
  File "/home/travis/.local/lib/python3.9/site-packages/qtpyvcp/actions/__init__.py", line 61, in bindWidget
    method = getattr(method, item)
AttributeError: module 'qtpyvcp.actions' has no attribute 'settings'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/travis/.local/lib/python3.9/site-packages/qtpyvcp/widgets/form_widgets/main_window.py", line 185, in getMenuAction
    actions.bindWidget(menu_action, action_name)
  File "/home/travis/.local/lib/python3.9/site-packages/qtpyvcp/actions/__init__.py", line 66, in bindWidget
    raise InvalidAction("Could not get action method: %s" % item)
qtpyvcp.actions.InvalidAction: Could not get action method: settings
2022-07-26 10:53:48,846 - qtpyvcp.app.launcher - INFO - Loading POSTGUI_HALFILE: /home/travis/linuxcnc/configs/my_LinuxCNC_router/postgui_call_list.hal
2022-07-26 10:54:34,433 - qtpyvcp.actions.machine_actions - INFO - Homing all axes
2022-07-26 10:55:03,064 - qtpyvcp.actions.machine_actions - INFO - Homing all axes
2022-07-26 10:55:28,076 - qtpyvcp.widgets.button_widgets.subcall_button - ERROR - Subroutine file could not be found: go_to_zero.ngc
2022-07-26 10:56:13,181 - qtpyvcp.widgets.input_widgets.file_system - INFO - File encoding: ascii
2022-07-26 10:56:14,490 - qtpyvcp.widgets.input_widgets.gcode_text_edit - INFO - File encoding: ascii
2022-07-26 10:56:15,347 - qtpyvcp.widgets.input_widgets.gcode_text_edit - INFO - File encoding: ascii
